bool display_map
time call_time
time position_time
float64 position_x
float64 position_y
